formbricks/formbricks
# fix/invite flow

## What does this PR do?



Fixes the check for expiration date on invite link visit and changes pending to expired on the settings page.



![image](https://github.com/formbricks/formbricks/assets/42251569/e10effb8-2a40-4630-b8ba-d3426eca69d7)
![image](https://github.com/formbricks/formbricks/assets/42251569/4c83f5f1-eebe-4f97-be0c-62bec379342b)

## Type of change




## How should this be tested?



- Open expired invitation link when logged in
- Open expired invitation link when logged out
- Visit settings page and see if pending changes to expired

## Checklist
